variables.tf

```diff
@@ -99,3 +99,12 @@ variable "consent_stores" {
   description = "Datastore that contain all information related to the configuration and operation of the Consent Management API (https://cloud.google.com/healthcare/docs/how-tos/consent-managing)."
   default     = []
 }
+
+# Extra fields for workspaces:
+#  settings: object (required)
+#    data_project_ids: list(string)
+variable "workspaces" {
+  type        = any
```

Comment:
@sana-google can u plz define this as an object instead of type `any`

Reply:
Done.

@yeweidaniel in all the existing definitions, `any` has been used, just FYI. For now I have updated this for the `workspaces` variable.